home/src/main/resources/rdf/accessControl/firsttime/profile_proximity_query.n3

```diff
@@ -70,8 +76,81 @@ access-individual:PersonProfileProximityToResourceUri a access:SparqlSelectValue
         }
         UNION
         { 
-          BIND ( ?profileUri as ?resourceUri)
+          BIND (?profileUri as ?resourceUri)
         }
     }
     """ .
 
+access-individual:ExternalIdMatchProfileProximityToResourceUri a access:SparqlSelectValuesQuery ;
+    access:id """
+    SELECT ?resourceUri WHERE {
+        ?profile ?matchingPropertyUri ?externalAuthId .
+        {
+          ?profile <http://purl.obolibrary.org/obo/RO_0000053> ?roleUri .
+          ?roleUri a <http://vivoweb.org/ontology/core#AdvisorRole> .
+          ?roleUri <http://vivoweb.org/ontology/core#relatedBy> ?resourceUri .
+          ?resourceUri a <http://vivoweb.org/ontology/core#AdvisingRelationship> .
+        }
+        UNION
+        {
+          ?profile <http://purl.obolibrary.org/obo/RO_0000053> ?roleUri .
+          ?roleUri a <http://vivoweb.org/ontology/core#TeacherRole> .
+          ?roleUri <http://purl.obolibrary.org/obo/BFO_0000054> ?resourceUri .
+          ?resourceUri a <http://vivoweb.org/ontology/core#Course> .
+        }
+        UNION
+        {
+          ?profile <http://purl.obolibrary.org/obo/RO_0000053> ?roleUri .
+          ?roleUri a <http://vivoweb.org/ontology/core#PrincipalInvestigatorRole> .
+          ?roleUri <http://vivoweb.org/ontology/core#relatedBy> ?resourceUri .
+          ?resourceUri a <http://vivoweb.org/ontology/core#Grant> .
+        }
+        UNION
+        {
+          ?profile <http://purl.obolibrary.org/obo/RO_0000053> ?roleUri .
+          ?roleUri a <http://vivoweb.org/ontology/core#CoPrincipalInvestigatorRole> .
+          ?roleUri <http://vivoweb.org/ontology/core#relatedBy> ?resourceUri .
+          ?resourceUri a <http://vivoweb.org/ontology/core#Grant> .
+        }
+        UNION
+        {
+          ?profile <http://vivoweb.org/ontology/core#relatedBy> ?roleUri .
+          ?roleUri a <http://vivoweb.org/ontology/core#Authorship> .
+          ?roleUri <http://vivoweb.org/ontology/core#relates> ?resourceUri .
+          ?resourceUri a <http://purl.obolibrary.org/obo/IAO_0000030> .
+        }
+        UNION
+        {
+          ?profile <http://vivoweb.org/ontology/core#relatedBy> ?roleUri .
+          ?roleUri a <http://vivoweb.org/ontology/core#Editorship> .
+          ?roleUri <http://vivoweb.org/ontology/core#relates> ?resourceUri .
+          ?resourceUri a <http://purl.obolibrary.org/obo/IAO_0000030> .
+        }
+        UNION
+        {
+          ?profile <http://purl.obolibrary.org/obo/RO_0000053> ?roleUri .
+          ?roleUri a <http://vivoweb.org/ontology/core#PresenterRole> .
+          ?roleUri <http://purl.obolibrary.org/obo/BFO_0000054> ?resourceUri .
+          ?resourceUri a <http://vivoweb.org/ontology/core#Presentation> .
+        }
+        UNION
+        {
+          ?profile <http://purl.obolibrary.org/obo/RO_0000053> ?roleUri .
+          ?roleUri a <http://vivoweb.org/ontology/core#ClinicalRole> .
+          ?roleUri <http://vivoweb.org/ontology/core#contributesTo> ?resourceUri .
```

Comment:
Should this be:
```suggestion
          ?roleUri <http://vivoweb.org/ontology/core#relates> ?resourceUri .
```

I can't find #contributesTo in the VIVO ontology

Reply:
Fixed property uri to roleContributesTo